src/ocp_postprocess/hostname_rename/etcd_rename.rs

```diff
@@ -387,6 +387,32 @@ pub(crate) async fn fix_etcds_cluster(etcd_client: &Arc<InMemoryK8sEtcd>, hostna
     Ok(())
 }
 
+pub(crate) async fn fix_node_name(etcd_client: &Arc<InMemoryK8sEtcd>, original_hostname: &str, hostname: &str) -> Result<()> {
+    let mut k8s_resource_location = K8sResourceLocation::new(None, "Minion", original_hostname, "v1");
```

Comment:
Let's change the [plural](https://github.com/rh-ecosystem-edge/recert/blob/dc0186f23a561ba65e1a399a9a383775d9840cb0/src/cluster_crypto/locations.rs#L54) to minions so we can do this:

```suggestion
    let mut k8s_resource_location = K8sResourceLocation::new(None, "Node", original_hostname, "v1");
```